### PR TITLE
games-strategy/freeorion: add 0.4.10 and fixes

### PR DESCRIPTION
**Pull request by cengique**
For 0.4.10, clone git release branch instead of downloading broken release.
For all ebuilds, changed eclass cmake-util to cmake as repoman
suggested.

Closes: https://bugs.gentoo.org/735768

Package-Manager: Portage-3.0.4, Repoman-3.0.1

@NP-Hardass

### CODE REVIEW

**Comment by cengique** [comment]
> ## Missing GCO sign-off
> 
> Please read the terms of [Gentoo Certificate of Origin](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin) and acknowledge them by adding a sign-off to _all_ your commits.

Uh, I wish this was mentioned somewhere in https://wiki.gentoo.org/wiki/Gentoo_GitHub#Commit_format

**Comment by DarthGandalf** [comment]
You're right! Added it to https://wiki.gentoo.org/wiki/Gentoo_GitHub#Commit_format

File: games-strategy/freeorion/freeorion-0.4.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+inherit cmake python-single-r1 xdg
+
+DESCRIPTION="A free turn-based space empire and galactic conquest game"
+HOMEPAGE="https://www.freeorion.org"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/freeorion/freeorion.git"
+else
+	if [[ ${PV} = *_p* ]]; then
```
Comment (DarthGandalf) [inline games-strategy/freeorion/freeorion-0.4.10.ebuild:16]: I think this was a hack specifically for `0.4.9_p…` because the commit is hardcoded here.

**Comment by o01eg** [comment]
> For 0.4.10, clone git release branch instead of downloading broken release.

It was fixed in 0.4.10.1

**Comment by cengique** [comment]
Fixed issues @o01eg and @DarthGandalf . But @NP-Hardass already committed ccd85b73ed903bd8696a15ca83a70b6b114dc96d with similar changes. Close this RP?

**Pull request closed without merging**